client/src/components/ui/select.tsx

```diff
@@ -15,7 +15,8 @@ export const triggerVariants = cva(
     variants: {
       variant: {
         default: '',
-        filter: 'data-[state=open]:border-gray-400',
+        'dataset-date': 'data-[state=open]:border-gray-400',
+        'project-date': 'data-[state=open]:border-gray-400',
```

Comment:
It seems that those two variants are always exactly the same. Maybe we could have just one instead called `'datepicker'` or something?